cocoa/src/toga_cocoa/widgets/progressbar.py

```diff
@@ -4,32 +4,65 @@
 
 from .base import Widget
 
+# Implementation notes
+# ====================
+#
+# * Cocoa doesn't have a way to determine if the animation is running. We track
+#   running status independently for interface compliance.
+
 
 class ProgressBar(Widget):
     def create(self):
         self.native = NSProgressIndicator.new()
         self.native.style = NSProgressIndicatorBarStyle
         self.native.displayedWhenStopped = True
+        self.native.usesThreadedAnimation = False
 
         # Add the layout constraints
         self.add_constraints()
 
+        self._is_running = False
+
+    def get_enabled(self):
+        # An activity indicator is always enabled
+        return True
```

Comment:
The interface layer overrides the `enabled` property, so I don't think this will ever be called.

Reply:
Ah - I thought I got them all. I'll purge this one.